bundle/src/assembly/resources/fed/ns3/ns3_installer.sh

```diff
@@ -45,7 +45,7 @@ arg_federate_file=""
 arg_integration_testing=false
 arg_make_parallel=""
 
-required_programs=( python3 gcc unzip tar )
+required_programs=( python3 gcc unzip tar lbzip2 protobuf-compiler )
```

Comment:
why `lbzip2`?

Reply:
the `unzip` wasn't able to extract the `tar.bz2` file, at least in my case

Reply:
and (in a different task) for extracting the `.tar.bz2` with `tar` I had to install `bzip2` which was also missing in wsl